certification/internal/bindings/example.go

```diff
@@ -0,0 +1,18 @@
+package bindings
+
+// import (
+// 	"fmt"
+// )
+
+// func TestListImages() {
```

Comment:
Can you go ahead and remove this file, and instead just add the example to a comment on this PR?

Reply:
this has been done. Been added to the body of the PR.